app/views/hyrax/dashboard/sidebar/_bulkrax_sidebar_additions.html.erb

```diff
@@ -1,10 +1,14 @@
 <% if current_ability.can_import_works? %>
-  <%= menu.nav_link(bulkrax.importers_path) do %>
+  <%= menu.nav_link(bulkrax.importers_path,
+                    alt: t('bulkrax.admin.sidebar.importers'),
```

Comment:
I'm unclear why we have both `alt` and `title` tag?  This element looks to render as a link, without an image.  Thus `title` should be adequate.  If it were an image, then we would want an `alt` tag.

https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/alt